clang/bindings/python/pyproject.toml

```diff
@@ -0,0 +1,36 @@
+[build-system]
+requires = ["setuptools>=42", "setuptools_scm==8.1.0"]
```

Comment:
Is there a reason you've pinned to setuptools_scm exactly, but not setuptools? I would generally avoid this, unless there's a really good reason to pin it.